### PR TITLE
Create script/server

### LINKED ISSUE #28: req.flash() requires sessions
The flash-connect middle wear requires use of express-sessions. Although session is initialized in server.js, an issue is raided when the sessions controller checks for potential flash messages:
```javascript
  if (req.flash('error')) {
    res.render('login', {message: req.flash('error')});
  } else {
    res.render('login'); 
  }
```
When that code is removed, we can proceed with normal login

### PR DESCRIPTION
**Pull request by tarebyte**
Closes #28 (this will close the issue when the PR is merged :octocat:)

@atareshawty I made some changes, see what you think.

The problem with running redis in the background is that you can forget to kill the server and then you have to grep through and kill it in order to restart it.

### CODE REVIEW

**Comment by tarebyte** [comment]
@atareshawty I updated `script/server` to use [ddollar/forego](https://github.com/ddollar/forego), it's an implementation of [foreman](https://github.com/ddollar/foreman) in [go](https://golang.org/)

**Comment by tarebyte** [comment]
This is what `heroku local` was running but I think using the actual command makes it more straight forward

**Comment by atareshawty** [comment]
I like this solution. If more server setup needs done, this will make it easier

**Comment by atareshawty** [comment]
This doesn't work with a redis-server already running. Can forego be used to check to see if it's already running?

File: script/server [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+# script/server: Launch the application and any extra required processes
+#                locally.
+
+set -e
+
+cd "$(dirname "$0")/.."
+
+if ps x |grep -v grep |grep -c redis-server; then
```
Comment (atareshawty) [inline script/server]: @tarebyte This seems to work as needed. I did npm start with redis running and without it running and the result was the same. I think it's good to go, but I'll wait for your opinion

**Pull request merged**